packages/vanilla/src/components/footer/footer.template.js

```diff
@@ -0,0 +1,87 @@
+import footerNav from './footerNav.template';
+import { Accordion } from 'carbon-components';
+import { settings } from 'carbon-components';
+
+const { prefix } = settings;
+
+/**
+ * renders accordion components for footer on smaller screen sizes
+ *
+ * @param {string} El type of footer in use
+ * @returns {object} JSX object
+ */
+function footerCreate(El) {
+  const accordionEl = El.querySelector('.bx--accordion');
+  if (accordionEl) {
+    // eslint-disable-next-line
+    const accordionInit = Accordion.create(accordionEl);
+  }
+}
```

Comment:
I'm wondering if we should pull the initialize code into its own file, and keep the templates in the `.template.js` files. We should probably follow that model I think especially when we hit more complex funcitonality like in the Masthead. We can decide if we want to create as an ES6 class with an init, or maybe something simpler like this